crawler/crawler/gitlab.go

```diff
@@ -36,30 +36,26 @@ func RegisterGitlabAPI() OrganizationHandler {
 			headers["Authorization"] = token
 		}
 
-		u, err := url.Parse(link)
-		if err != nil {
-			return link, err
-		}
 		// Set domain host to new host.
-		domain.Host = u.Hostname()
+		domain.Host = url.Hostname()
 
-		apiURL, _ := u.Parse("/api/v4")
+		apiURL, _ := url.Parse("/api/v4")
```

Comment:
**[suggestion]** You could make /api/v4 a constant.

Reply:
This one will be refactored in a following patch tackling https://github.com/italia/developers-italia-backend/issues/257.

We should fully rely on the code hosting's libraries instead of doing direct HTTP calls.